src/plugins/wb-data-scrub/wb-data-scrub.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * @title WET-BOEW wb-data-scrub
+ * @overview This plugin delete Personal Identifiable Information (PII) from the flagged form fields before form submit
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @polmih, @duboisp, @GormFrank
+ **/
+( function( $, wb ) {
+"use strict";
+
+var $document = wb.doc,
+	componentName = "wb-data-scrub",
+	selector = "[data-rule-wbscrub]",
+	initEvent = "wb-init" + componentName,
+	wrapper,
+
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector );
+
+		if ( elm && $( elm.form ).data( "scrub-inited" ) !== "" ) {
+
+			var	form = elm.form;
+
+			$( elm.form ).data( "scrub-inited", "" );
+
+			form.addEventListener( "submit", function( event ) {
+				event.preventDefault();
+				var isPii = false,
+					moDalId = componentName + "-modal",
+					i18nDict = {
```

Comment:
No need to define the i18nDict, instead retrieve it from wb.

Something like: https://github.com/wet-boew/wet-boew/blob/master/src/plugins/multimedia/multimedia.js#L108-L120